### PR TITLE
feat(console): add next server actions guide

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add next server actions guide based on https://blog.logto.io/nextjs-server-actions/

## Testing

Local preview

截屏2023-12-26 上午11 37 22

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

File: packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+pnpm add @logto/next
+```
+
+</TabItem>
+</Tabs>
+</Step>
+
+<Step
+  title="Init LogtoClient"
+  subtitle="2 step"
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx:42]: we can remove those subtitles with `* steps`

File: packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+Add the following code to `libraries/logto.js`:
+
+```ts
+export const getLogtoContext = async () => {
+  return await logtoClient.getLogtoContext(getCookie());
+};
+```
+
+### Fetch in home page
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx:235]:
```suggestion
### Fetch authentication data
```

File: packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+const SignOut = () => {
+  const router = useRouter();
+
+  const handleClick = async () => {
+    const redirectUrl = await signOut();
+
+    router.push(redirectUrl);
+  };
+
+  return <button onClick={handleClick}>Sign Out</button>;
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx:328]:
```suggestion
  return <button onClick={handleClick}>Sign out</button>;
```

File: packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx [hunk 4]
```diff
@@ -0,0 +1,355 @@
...
+const SignIn = () => {
+  const router = useRouter();
+
+  const handleClick = async () => {
+    const redirectUrl = await signIn();
+
+    router.push(redirectUrl);
+  };
+
+  return <button onClick={handleClick}>Sign In</button>;
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx:162]:
```suggestion
  return <button onClick={handleClick}>Sign in</button>;
```

File: packages/console/src/assets/docs/guides/web-next-server-actions/index.ts [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import { ApplicationType } from '@logto/schemas';
+
+import { type GuideMetadata } from '../types';
+
+const metadata: Readonly<GuideMetadata> = Object.freeze({
+  name: 'Next.js (Server Actions)',
+  description:
+    "Next.js with Server Actions, take advantage of async component of React's latest features.",
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/web-next-server-actions/index.ts:8]: "take advantage of" sounds weird - how about

```suggestion
    "Next.js with Server Actions, leverages async component from the latest features of React.",
```

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+  appId: '${props.app.id}',
+  appSecret: '${props.app.secret}',
+  baseUrl: 'http://localhost:3000', // Change to your own base URL
+  cookieSecret: '${generateStandardSecret()}', // Auto-generated 32 digit secret
+  cookieSecure: process.env.NODE_ENV === 'production',
+};
+
+export const logtoClient = new LogtoClient(config);`}
+  </code>
+</pre>
```
Comment (darcyYe) [inline packages/console/src/assets/docs/guides/web-next-server-actions/README.mdx:69]: nit: why not align the use of code block

**Review by darcyYe — APPROVED** [review]

**Pull request merged**